test/test_python_dispatch.py

```diff
@@ -2002,6 +2002,8 @@ def __new__(cls, data, wrapper):
                 def __torch_dispatch__(cls, func, types, args, kwargs):
                     if func.overloadpacket == torch.ops.aten.is_contiguous:
                         return contiguous_data.is_contiguous()
+                    if func.overloadpacket == torch.ops.aten.sym_is_contiguous:
+                        return torch.ops.aten.sym_is_contiguous(contiguous_data)
```

Comment:
Hmm, I'm a little uncertain about this. If you are actually forcing all torch dispatch classes to add handling for `sym_is_contiguous` this is BC breaking. When there aren't symbolic integers anywhere in the Tensor, it would be better for BC if we still route to `is_contiguous`. I'm unsure how inconvenient this is, so I can be talked out of it, but needing to change it here is a smell.